init/mesosite.sql

```diff
@@ -132,6 +132,30 @@ CREATE UNIQUE index iembot_twitter_subs_idx on
 GRANT ALL on iembot_twitter_subs to nobody;
 
 
+---
+--- IEMBOT Mastodon Page subscriptions
+---
+CREATE TABLE iembot_mastodon_oauth(
+  user_id bigint NOT NULL UNIQUE,
```

Comment:
Is `user_id` unique among the entire mastodon universe or only unique to the local mastodon instance?

Reply:
I think I misunderstood this field when I was hacking away on this last night, I thought `user_id` was internal to the IEM (though then it would probably `AUTO INCREMENT`...). I don't think a user's Mastodon `id` is unique across the Fediverse, so if you want a unique _public_ identifier, it would probably have to be a combo of the `id` and `api_base_url`.